src/Domain/Common/CommonEntity.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace CleanArchitecture.Domain.Common
+{
+    public class CommonEntity : AuditableEntity
+    {
+        public int Id { get; set; }
+        public string Title { get; set; }
```

Comment:
I wouldn't add a property such as `Title` since it wouldn't be used by all entities in most systems.